integration-tests/src/test/java/tachyon/client/keyvalue/KeyValuePartitionIntegrationTest.java

```diff
@@ -76,4 +78,52 @@ public void readerWriterTest() throws IOException, TachyonException {
     Assert.assertArrayEquals(VALUE2, mKeyValuePartitionReader.get(KEY2));
     Assert.assertNull(mKeyValuePartitionReader.get("NoSuchKey".getBytes()));
   }
+
+  @Test
+  public void sizeTest() throws Exception {
+    byte[][] keys = new byte[][]{KEY1, KEY2};
```

Comment:
Readability, especially since using an array leaves us with a double-nested array. `new byte[][]{VALUE1, VALUE2}` looks really weird to me. It's not many more characters to do `Lists.newArrayList(VALUE1, VALUE2)`